### DISCUSSION TITLE
Can no longer resize dialog boxes

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1957] [General]
I thought it was possible to resize dialog boxes but all of a sudden I am noticing that no dialog boxes resize.  For example the Create New Directory/Folder dialog cannot.
That one might not have been able to be resized in the past but I know Lua script based dialog boxes were able to, now they do not either.
I was wondering if anyone knows why this might be occurring?
Or if anyone can confirm that this is happening on their machine as well?

Thank You

Double Commander 
1.2.0 alpha
Revision 11533
Commit 6a99fd2eb
Lazarus 3.0.0.3
Free Pascal  3.2.2
QT5
Standalone Version
Debian based Linux distro / GTK based Desktop Environment

**Comment by ags1234** [comment]
Double Commander
Version: 1.2.0 alpha
Revision: 11604
Commit: c199a7d36
Build date: 2024/10/22
Lazarus: 3.6.0.0
FPC: 3.2.2
Platform: i386-Win32-win32/win64
OS version: Windows 11 23H2 x86_64

Create new directory dialog does not resize.
Copy/Move can be resized, even too much I would say :D

**Comment by j2969719** [comment]
dialogboxes (BorderStyle = bsDialog) cannot be resized
the copy/move window is not a dialogbox for some reason

**Comment by JonnyTech** [comment]
> QT5
> Standalone Version
> Debian based Linux distro / GTK based Desktop Environment


Is it the same on GTK2 version?

**Comment by advice2020** [comment]
Thank you guys for the responses.
Cannot say, I am not using the GTK2 versions.

I tried downloading current official QT5 AppImage release and same results.
Lua script dialog boxes no longer can change size?
I remember when I had dialog issues due to Lazarus bug where it chopped off button edges, I was able to resize Lua script dialog boxes.
So it was possible in the past at least with Lua dialogs.
Does anyone know if this ability to resize Lua script generated dialog boxes has been removed?
or could this be a regression bug?

Thank You

**Comment by advice2020** [comment]
@alexx2000
I apologize I do not mean to call you specifically, but do you by any chance know if Lua script input field dialog box code has been changed to no longer support adjustable resizing?

Thank You

**Reply by alexx2000** [reply]
As I remember, no. Maybe Lazarus changed. Technically all dialogs has `bsSingle` or `bsDialog` style which does not support resizing.
